src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -1159,6 +1168,35 @@ class AddItemsPage extends Component {
 
     return Promise.resolve();
   }
+  saveRequisitionItemsInCurrentStepWithAlert({
+    lineItems,
+    spinner = false,
+    callback = () => {},
+  }) {
+    confirmAlert({
+      title: this.props.translate('react.stockMovement.message.confirmSave.label', 'Confirm save'),
+      message: this.props.translate(
+        'react.stockMovement.QOHWillNotBeSaved.message',
+        'This save action won’t save the quantity on hand you have entered. You will have to reenter these when you came back to this request later. Are you sure you want to proceed?',
+      ),
+      buttons: [
+        {
+          label: this.props.translate('react.default.yes.label', 'Yes'),
+          onClick: () => {
+            if (spinner) this.props.showSpinner();
```

Comment:
Why it is dependant on the `spinner` flag?

Reply:
because previously spinner was not called in **saveAndExit()** and **previousePage()** when calling **saveRequisitionItemsInCurrentStep**

Reply:
@DaRoTP It should always be triggered when there is the save call. Please remove this flag.